src/components/BookComponents/Authors/Authors.jsx

```diff
@@ -0,0 +1,24 @@
+import { printAuthors } from "../../../helpers/helpers";
+import styles from "./Authors.module.scss";
+
+const Authors = ({ authors, isList }) => {
+  // authors && isList
+  // - authors ? and more : [0]
+  // no authors || no list
+  // - authors ? printAu : null
+  return (
+    <>
+      {authors && isList ? (
+        authors.length > 1 ? (
+          <p className={styles.details__authors}>{` ${authors[0]} and more`}</p>
+        ) : (
+          <p className={styles.details__authors}>{authors[0]}</p>
+        )
+      ) : authors ? (
+        <p className={styles.details__authors}>{printAuthors(authors)}</p>
+      ) : null}
```

Comment:
If you clean the book data in a seperate function you can write less logic in your components. Will make them easier to read. But I do like that you split the logic into a seperate component so your book content and book modal were clean.